.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseDateTimePeriodParser.cs

```diff
@@ -468,6 +468,19 @@ protected virtual DateTimeResolutionResult ParseSpecificTimeOfDay(string text, D
                     ret.Comment = "late";
                 }
             }
+            else
+            {
+                match = this.Config.AmDescRegex.Match(trimedText);
```

Comment:
How are you sure this only applies to the right case?
In the issue, AM or PM would appear after a DATE, not a SpecificTimeOfDay

Reply:
Function ParseSpecificTimeOfDay() is to parse the specific time of day appears around a DATE.